ui/partials/perf/graphsctrl.html

```diff
@@ -74,9 +74,25 @@
       </div>
       <div>
         <p ng-show="tooltipContent.revision">
+          <button title="Nudge left"
+            ng-if="user.isStaff && tooltipContent.alert"
+            ng-click="nudgeAlert(tooltipContent, 'left')">
```

Comment:
@wlach Can you look into this nudgeAlert() function? Is tooltipContent the right parameter for it? Somehow, when I want to update an alert using the id I find in tooltipContent, the BE throws a 404.